omniauth-greenhouse.gemspec

```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.3'
   spec.add_development_dependency 'rake'
 
-  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1.1.1'
-  spec.add_runtime_dependency 'omniauth', '~> 1.2.1'
+  spec.add_runtime_dependency 'omniauth-oauth2', '~> 1'
+  spec.add_runtime_dependency 'omniauth', '>= 1.3.1', '< 2'
```

Comment:
@tfrey7 The issue we're trying to fix is fixed in `omniauth` 1.3.1, but `omniauth-oauth2`'s latest release (1.4) has a minor version requirement on `omniauth` ~> 1.2.  There is an [outstanding issue](https://github.com/intridea/omniauth-oauth2/issues/87) to upgrade to 1.3.  I think we should add:

```
spec.add_runtime_dependency 'omniauth', '>= 1.3.1', '< 2'
```

to ensure that 1.3.1 is being used.